agent/01_agent_requirements.sh

```diff
@@ -50,3 +60,16 @@ if [[ "${AGENT_E2E_TEST_BOOT_MODE}" == "ISCSI" ]]; then
     # Install shell to administer local storage
     sudo dnf -y install targetcli
 fi
+
+if [[ "${AGENT_E2E_TEST_BOOT_MODE}" == "ISO_NO_REGISTRY" ]]; then
+   sudo dnf -y install xorriso coreos-installer syslinux
```

Comment:
Since iso_builder requires `skopeo` this should also be included, otherwise get:
`/home/stack/go/src/github.com/openshift/agent-installer-utils/tools/iso_builder/hack/helper.sh: line 90: skopeo: command not found`